packages/js/data/src/crud/action-types.ts

```diff
@@ -9,6 +9,8 @@ export enum TYPES {
 	GET_ITEMS_SUCCESS = 'GET_ITEMS_SUCCESS',
 	UPDATE_ITEM_ERROR = 'UPDATE_ITEM_ERROR',
 	UPDATE_ITEM_SUCCESS = 'UPDATE_ITEM_SUCCESS',
+	GET_ITEMS_TOTAL_COUNT_SUCCESS = 'GET_ITEMS_TOTAL_COUNT_SUCCESS',
+	GET_ITEMS_TOTAL_COUNT_ERROR = 'GET_ITEMS_TOTAL_COUNT_ERROR',
```

Comment:
All the changes within the `crud` data store revolve around adding the `getItemsTotalCount` support, it should be more or less self explanatory.